test/Microsoft.Azure.SignalR.Tests/MultiEndpointServiceConnectionContainerTests.cs

```diff
@@ -1050,8 +1050,8 @@ public async Task TestEndpointManagerWithAddEndpointsWithServersTag()
             var serversTag = "Server1;Server2;Server3";
             await Task.WhenAll(containers.Select(c => c.MockReceivedServersPing(serversTag)));
 
-            // wait one interval+ for Ready state and check negotiation is added.
-            await Task.Delay(6000);
+            await hubEndpoints[1].ScaleTask;
```

Comment:
await hubEndpoints[1].ScaleTask.OrTimeout(10000) ?

Reply:
`ScaleTask` is completed, but other operations need a few moment delay before negotiation endpoints get updated. I think `OrTimeout()` means the longest time that should be await when target task doesn't complete as expected.

In normal case it takes about 5.2s of an interval. Do you mean add `OrTimeout()` to make it fail fast in error case?